FreeRTOS/FreeRTOS
# Build: Fix Demo/RX600_RX64M_RSK_GCC_e2studio RTOS demo project build error

Description
-----------
This PR fixes Demo/RX600_RX64M_RSK_GCC_e2studio RTOS demo project build error which is caused by the same reason as follows.

RSK64M Demo - Fix build errors #195
Build: Make Demo/RX600_RX64M_RSK_Renesas_e2studio RTOS demo project better #231

This PR has no changes in the runtime functionality.

Test Steps
-----------
I did the following steps on the e2 studio 3.1.2.09 and KPIT GNURX v14.01. (According to .info file, the project was originally built with the same versions of the tools.)

(1) Import the project
(2) Clean all projects and start a build

But I don't have the evaluation board therefore I can't check the program which are built.
If this isn't acceptable for you, please let me know.

Related Issue
-----------
[BUG] Renesas RSK64F Demo build errors (linker) #42
Resolve stdio and stream_buffer link errors in RX64M demo (#42) #43

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
